websites/D/Dash Radio/dist/metadata.json

```diff
@@ -0,0 +1,20 @@
+{
+  "author": {
+    "name": "apprehends",
+    "id": "202236922217758721"
+  },
+  "service": "Dash Radio",
+  "description": {
+    "en": "100% Free Music. More Variety. Zero Commercials. The Best DJ's & Stations. No BS"
+  },
+  "url": "dashradio.com",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/o4Ea2f9.png",
+  "thumbnail": "https://i.imgur.com/8612XSi.png",
+  "color": "#55effa",
```

Comment:
thumbnail should be a promotional banner or a screenshot of the website homepage.

Reply:
> Upload new proof of it working

https://streamable.com/wqyoyn